### PR TITLE
docs-infra - implemement `@reviewed` tag

### PR DESCRIPTION
**Pull request by petebacondarwin**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by aikithoughts — APPROVED** [review]
This is great! Thank you!

Reviewed-for: global-docs-approvers

File: aio/src/styles/2-modules/_guides.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.reviewed {
+  font-style: italic;
```
Comment (gkalpak) [inline aio/src/styles/2-modules/_guides.scss:2]: Nit: Personally, I would make it a little more clear that this is different from the main content by moving it to the bottom-left and giving it s slightly smaller font and grayer color:

```suggestion
  color: lighten($darkgray, 10);
  display: block;  // Or better yet replace the `span` with a block elements.
  @include font-size(13);
  font-style: italic;
  text-align: right;
```

reviewed

**Review by gkalpak — APPROVED** [review]
Nice!

File: aio/tools/transforms/content-package/tag-defs/reviewed.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ * The date-string will be passed to the `Date()` constructor.
+ */
+module.exports = function() {
+  return {
+    name: 'reviewed',
+    transforms(doc, tag, value) {
+      return {
+        date: new Date(value),
+      };
+    }
```
[the remark this answers is not in the document]
Comment (petebacondarwin) [reply aio/tools/transforms/content-package/tag-defs/reviewed.js:14]: Yes me too

File: aio/src/styles/2-modules/_guides.scss (hunk 1 above)
Comment (petebacondarwin) [reply aio/src/styles/2-modules/_guides.scss:2]: Also works for me

**Pull request closed without merging**